# common params
run_factor: 2.0

# buttons
lower_neck_button: 6
upper_neck_button: 4
tray_button: 3
arm_joint12_button: 0
arm_joint34_button: 1
arm_joint56_button: 2
arm_joint7_button: 3
deadman_button: 5
run_button: 7

# axes
axis_vx: 1
axis_vy: 0
axis_vth: 2
up_down: 5 #tray--up/down; tilt--front/back, here we just name up_down
left_right: 4 #pan--left/right

# velocities in rad/sec, m/sec
lower_tilt_step: 0.05
lower_pan_step: 0.05
upper_tilt_step: 0.075
upper_pan_step: 0.075
tray_step: 0.15
arm_left_right_step: 0.1
arm_up_down_step: 0.1
max_vx: 0.3
max_ax: 0.5
max_vy: 0.2
max_ay: 0.5
max_vth: 0.3
max_ath: 0.7
